docs/source/posts/2023/2023-06-05-week-1-joaodellagli.rst

```diff
@@ -0,0 +1,61 @@
+The FBO Saga - Week 1
+=====================
+
+.. post:: June 5, 2023
+   :author: João Victor Dell Agli Floriano
+   :tags: google
+   :category: gsoc
+
+
+This Past Week
+--------------
+
+As mentioned in the last week's blogpost, the goal for that week was to investigate VTK's Framebuffer Object framework.
+An update on that is that indeed, VTK has one more low-level working `FBO class <vtk.org/doc/nightly/html/classvtkOpenGLFramebufferObject.html>`_ that can be used inside FURY, however, 
```

Comment:
The link does not work for me. Try using this one instead https://vtk.org/doc/nightly/html/classvtkOpenGLFramebufferObject.html

Reply:
Yeah it doesn't work for me as well!